action.yaml

```diff
@@ -368,6 +376,15 @@ runs:
         DAGS_ONLY_DEPLOY=false
         SKIP_IMAGE_OR_DAGS_DEPLOY=true
 
+        # case when the triggered event is a manual workflow dispatch, we would need to deploy the image because we cannot determine that it does not need to be deployed
+        GITHUB_EVENT_BEFORE=${{ github.event.before }}
+        GITHUB_EVENT_AFTER=${{ github.event.after }}
+        if [[ -z $GITHUB_EVENT_BEFORE && -z $GITHUB_EVENT_AFTER ]]; then
```

Comment:
nit: could we club both the `if` clause (this and the one below) into a single clause to avoid code repetition :)

Reply:
Addressed